pkg/rid/application/isa_test.go

```diff
@@ -34,15 +33,15 @@ func (store *isaStore) GetISA(ctx context.Context, id dssmodels.ID) (*ridmodels.
 	if isa, ok := store.isas[id]; ok {
 		return isa, nil
 	}
-	return nil, sql.ErrNoRows
+	return nil, nil
 }
 
 // DeleteISA deletes the IdentificationServiceArea identified by "id" and owned by "owner".
 // Returns the delete IdentificationServiceArea and all IdentificationServiceAreas affected by the delete.
 func (store *isaStore) DeleteISA(ctx context.Context, isa *ridmodels.IdentificationServiceArea) (*ridmodels.IdentificationServiceArea, error) {
 	isa, ok := store.isas[isa.ID]
 	if !ok {
-		return nil, sql.ErrNoRows
```

Comment:
Let's replace the comments in lines 39 & 40 with something like "Implements repos.ISA.DeleteISA" so we can avoid introducing inconsistencies into the comments like we have here.  Likewise for other implementations of interfaces.